web/src/stories/icon.stories.mdx

```diff
@@ -20,7 +20,7 @@ c.Config.Badge = component.BadgeDanger
 <Meta title="Components/Icon" component={IconComponent} />
 
 <Canvas withToolbar>
```

Comment:
Not sure the last `<p>` above is accurate any more, can't we remove Clarity 4 reference (same for signpost)?